DNN Platform/Connectors/GoogleAnalytics/GoogleAnalyticsConnector.cs

```diff
@@ -165,28 +173,27 @@ public bool SaveConfig(int portalId, IDictionary<string, string> values, ref boo
 
                 string trackingID;
                 string urlParameter;
-                bool trackForAdmin;
-                bool anonymizeIp;
-                bool trackUserId;
+                string trackForAdmin;
+                string anonymizeIp;
+                string trackUserId;
 
                 isValid = true;
 
                 if (isDeactivating)
                 {
                     trackingID = null;
                     urlParameter = null;
-                    trackForAdmin = false;
-                    anonymizeIp = false;
-                    trackUserId = false;
+                    trackForAdmin = null;
+                    anonymizeIp = null;
+                    trackUserId = null;
                 }
                 else
                 {
-                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToString().Trim() : String.Empty;
-                    urlParameter = values["UrlParameter"] != null ? values["UrlParameter"].ToString().Trim() : String.Empty;
-
-                    bool.TryParse(values["TrackAdministrators"].ToLowerInvariant(), out trackForAdmin);
-                    bool.TryParse(values["AnonymizeIp"].ToLowerInvariant(), out anonymizeIp);
-                    bool.TryParse(values["TrackUserId"].ToLowerInvariant(), out trackUserId);
+                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToLowerInvariant().Trim() : string.Empty;
```

Comment:
should tracking ID and URL parameter be lower-cased?

Reply:
Nope, the object is lower case but the key is Upper Case

Reply:
@valadas I'm asking about the `ToLowerInvariant()` call on the value, that shouldn't be there, should it?
```suggestion
                    trackingID = values["TrackingID"]?.Trim() ?? string.Empty;
```